.github/workflows/cli.yml

```diff
@@ -1,6 +1,9 @@
 name: Run Tests
 
 on:
+  push:
+    branches:
+      - master  
```

Comment:
Are we interested in running on merge to `v1` as well as `master`? Not a requirement for merge, just curious.

Reply:
Good question 🤔 I think so, I'll create another PR for that 👍